### ISSUE TITLE
audio type

### ISSUE THREAD

**Issue by eddilara** [yaser01/mkv-muxing-batch-gui#86]
it would be a good feature to be able to select which audio you want to keep based on the type of the audio
example Truehd/ac3/aac

**Comment by yaser01** [comment]
you are taking about old tracks ?
can't just this be same to keep track by id ?

**Comment by eddilara** [comment]
@yaser01 yes..but if lets say a series doesn't keep the same track in the same track number
then ep1 you are keeping the ac3 track...and for ep 12 you are keeping the dts track

**Comment by yaser01** [comment]
I see but what if video have multi ac3 tracks?
what will happen

**Comment by eddilara** [comment]
in that case you would probably end up using the language of the track if possible
since if theres 2 ac3 tracks...1 is probably a different language

**Comment by yaser01** [comment]
I see i will try to add this :D
